src/types.rs

```diff
@@ -88,6 +88,7 @@ impl DateTime {
     /// * hour: [0, 23]
     /// * minute: [0, 59]
     /// * second: [0, 60]
+    #[allow(clippy::result_unit_err)]
```

Comment:
A better error type will be nice when we can fix the api

Reply:
Sure. That's why I've commented it out for now :) Later, when we will rework an API, we definitely will fix it (at least I hope)